test/discr/discr_mappings_test.py

```diff
@@ -345,5 +346,19 @@ def test_collocation_interpolation_identity():
         assert all_almost_equal(ident_values, values)
 
 
+@skip_if_no_cuda
+def test_collocation_cuda():
+    rect = odl.Rectangle([0, 0], [1, 1])
+    part = odl.uniform_partition_fromintv(rect, [4, 2])
+    space = odl.FunctionSpace(rect)
+    dspace = odl.CudaRn(part.size)
+
+    coll_op = PointCollocation(space, part, dspace, order='F')
```

Comment:
Hm, coincidence I guess. Wasn't Fortran ordering the natural thing on CUDA spaces?